docs/project/consortium.md

```diff
@@ -0,0 +1,24 @@
+# DIRAC Consortium
```

Comment:
We will have the diracgrid.org page where these info will be displayed. Keep this documentation for diracx technicalities only (you can fully remove this one).